### ISSUE TITLE
Persistent 503 errors?

### ISSUE THREAD

**Issue by seancorfield** [clj-holmes/clj-watson#34]
For the last two or three days, I've been unable to run Watson:

```
Downloading/Updating database.
** ERROR: **
Exception: #error {
 :cause NVD Returned Status Code: 503
 :via
 [{:type org.owasp.dependencycheck.data.update.exception.UpdateException
   :message Error updating the NVD Data
   :at [org.owasp.dependencycheck.data.update.NvdApiDataSource processApi NvdApiDataSource.java 336]}
  {:type io.github.jeremylong.openvulnerability.client.nvd.NvdApiException
   :message NVD Returned Status Code: 503
   :at [io.github.jeremylong.openvulnerability.client.nvd.NvdCveClient next NvdCveClient.java 327]}]
 :trace
 [[io.github.jeremylong.openvulnerability.client.nvd.NvdCveClient next NvdCveClient.java 327]
```

I don't know whether this is genuinely due to some underlying service being down or whether it's a configuration issue (using a deprecated endpoint that has now been removed).

**Comment by seancorfield** [comment]
Looks like https://github.com/jeremylong/DependencyCheck/issues/6107

**Comment by seancorfield** [comment]
I've rolled back to 8.4.3 for now, but it sounds like the NVD stuff will _require_ users to switch to the new API key approach (and 9.0.1+ of DependencyCheck) fairly soon.

I requested an API key but I'm getting 403 errors now (with 9.0.1). I'm wondering if I have a bad API key. I'll revisit this if/when 8.4.3 stops working.

**Comment by seancorfield** [comment]
9.0.1 is not backward compatible with 8.4.3 so I suspect Watson will need changes to be able to work with 9.0.1 and let users provide their own API key (via the properties file, I think).

**Comment by seancorfield** [comment]
See #35 for some updates that make this work.

**Comment by seancorfield** [comment]
This will be addressed fully by #41

**Comment by seancorfield** [comment]
PR has been merged. Thank you!
